python/pyarrow/ipc.pxi

```diff
@@ -399,21 +399,23 @@ cdef class MessageReader(_Weakrefable):
                         "instead.".format(self.__class__.__name__))
 
     @staticmethod
-    def open_stream(source):
+    def open_stream(source, use_memory_map=False):
         """
         Open stream from source.
 
         Parameters
         ----------
         source
             A readable source, like an InputStream
+        use_memory_map : boolean, default False
+            Use memory mapping when opening file on disk
```

Comment:
```suggestion
```

Reply:
Let's add to the `source` docstring that you can pass `MemoryMappedFile` if you want to use memory mapping.